test/Sleep-test.js

```diff
@@ -66,7 +67,9 @@ describe('SleepRepository', () => {
   });
 
   it('should be able to filter the user"s data by ID', () => {
-    expect(sleepRepository.getSleepData()).to.deep.equal(
+    chai.spy.on(sleepRepository, 'getData', () => [{}]);
```

Comment:
Woohoo!  Nice work, Allison.  So exciting that we have a baby spy on our project.

Question (maybe I missed something during the lesson about the inheritance and spies) -- is it common to put a spy on one of our child classes?  I thought we put them on our parent classes?  Or we can do them in either case?

Tagging Brittany and Khalid here, too, just in case we don't push this into our master so they can see we dabbled in spies.

@brittanystoroz
@khalidwilliams